product/tests/test_serializers/test_item_serializer.py

```
@@ -0,0 +1,86 @@
+from decimal import Decimal
```

Ruff (0.8.2)

`decimal.Decimal` imported but unused

Remove unused import: `decimal.Decimal`

(F401)